news/2020-04-13-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+
+As such, we intend to create **a new competitive and seasonal league** based on spotlighted maps. With this, you and your friends will have even more reason to delve into some of the best creations that osu! has to offer.
+
+To accomplish this, a revamp of the Spotlights Team will be necessary. As such, we are now looking for anyone from the community that is both willing and capable of selecting the best of the best from the mapping scene. Those who sign up will be responsible for helping to build this new competitive league from the ground up.
+
+**If you think you have what it takes to be part of the new Spotlights Team and help shape a new form of competitive play from the very beginning, [apply today!](https://spotlights.team/app)** We are looking for a diverse mix of members, including (but not limited to) renowned mappers, accomplished modders, skilled players, mentors, tournament organizers and mappool selectors/creators.
+
+We're still working on the exact details of how this new league will work, but here's what we've got planned so far:
+
+- **We'll be hosting weekly timeshift lobbies created from Spotlight beatmap pools** using osu!lazer. Timeshift functions similarly to how beatmap charts currently do - if you have never heard of timeshift multiplayer lobbies before, [you should read peppy's article about the new multiplayer paradigm](https://blog.ppy.sh/enjoying-osu-with-others/), or just [hop into osu!lazer yourself and give it a try](https://github.com/ppy/osu/releases).
+- **Spotlights will be released incrementally in a weekly fashion instead of all at once in big seasonal releases**. As above, this means that **beatmaps being played in the Spotlights league lobbies will change every week**.
+- When a Spotlights timeshift lobby ends each week, **the winners will be awarded with a small prize**, likely including osu!supporter and other shiny things.
+- We're also experimenting with **a new ranking across each Spotlight season** where the results of each timeshift lobby will be collated and compiled into a custom-made ranking system. We plan to assign a league bracket to every participant, based on their placement in this ranking system (such as Bronze, Silver, Gold, etc.). Once a season ends, maybe you can even keep your placement in shape of a medal or badge on your profile...?
+
+The Beatmap Spotlights you know and love will still be here (including medals and commentary-packed newsposts), just with a big face-lift and hopefully a lot more interest from people at large.
+
+We hope that you like this new idea and that you'll consider helping shape the new Spotlights together with us. **[Join us at our Discord server](https://discord.gg/DtQrzyV)** or visit the `#osu-spotlights` channel of osu!dev to help us with this project going forwards!
```

Comment:
wouldn't make sense to linkify osu!dev as well?

Reply:
It wasn't linked in most past announcements either.